dev-docs/bidders/videobyte.md

```diff
@@ -5,15 +5,22 @@ description: Prebid VideoByte Bidder Adaptor
 pbjs: true
 pbs: true
 biddercode: videobyte
-media_types: video
+media_types: display, video
 gdpr_supported: true
 usp_supported: true
 schain_supported: true
 ---
 
 ### Note:
 
-VideoByte adapter only supports video ads.
+VideoByte Prebid Client adapter only supports video ads.
```

Comment:
@videobyte20 
On line 8 you have updated the field to say you support both display and video. I'm confused by this line.

Reply:
Display is supported when using Prebid Server, but not when using Prebid Client. How should we indicate this? Thanks.

Reply:
Suggestion:

"The VideoByte Prebid.js client adapter only supports video ads while our Prebid Server adapter supports both display and video."

Reply:
Updated with your suggestion, thanks!